checker/src/main/java/org/checkerframework/checker/initialization/InitializationAnnotatedTypeFactory.java

```diff
@@ -109,7 +113,7 @@
      * String representation of all initialization annotations.
      *
      * <p>{@link UnknownInitialization} {@link UnderInitialization} {@link Initialized} {@link
-     * FBCBottom}
+     * FBCBottom} {@link PolyInitialized}
      *
      * <p>This is used to quickly check of an AnnotationMirror is one of the initialization
      * annotations without having to repeatedly convert them to strings.
```

Comment:
I cannot comment on the field type, so I'll comment here: should the type be `Set<@CanonicalName String>` to make explicit that these are all fully-qualified names (looking at how the set is initialized)? Also, the javadoc should mention this.

Reply:
Ping on this.

Reply:
The set is initialized within the constructor at line 164. Already add a description in javadoc.